stdlib/config-api/src/main/java/org/ballerinalang/stdlib/config/SetConfig.java

```diff
@@ -65,4 +66,8 @@ public void execute(Context context) {
 
         context.setReturnValues();
     }
+
+    public static void setConfig(Strand strand, String configKey, Object configValue) {
```

Comment:
Again, what's the purpose of this method? As with the contains implementation, a `Strand` is passed as a param but not made use of.

Reply:
Strand is a first argument to any methods in ballerina. The impl may or may not use the strand argument based on the requirement. For example, if you want to perform a non-blocking invocation within the impl, then you may use the strand ref.